wrench/src/yaml_frame_writer.rs

```diff
@@ -311,21 +311,48 @@ impl YamlFrameWriter {
         scene.finish_root_display_list(self.pipeline_id.unwrap(), dl, aux);
 
         let mut root = new_table();
-        table_node(&mut root, "root", root_dl_table);
+        if let Some(root_pipeline_id) = scene.root_pipeline_id {
+
+            u32_vec_node(&mut root_dl_table, "id", &vec![root_pipeline_id.0, root_pipeline_id.1]);
+
+            let mut pipelines = vec![];
+            for pipeline_id in scene.pipeline_map.keys() {
+                // write out all pipelines other than the root one
+                if pipeline_id.0 == root_pipeline_id.0 && pipeline_id.1 == root_pipeline_id.1 {
```

Comment:
there's no PartialEq defined for this, can't just pipeline_id == root_pipeline_id?